win10-difference.cmd

```diff
@@ -0,0 +1,136 @@
+rem commit message comments:
+
+rem Need to 
+rem - install npm
+rem - install curl
+rem - install more packages
+rem - curl need to use double quote in the curl parameters
+rem - curl need to escape double quote in json string
+
+rem install curl exe
+rem install npm (node.js)
+
+rem update npm 
+
+npm i npm
+
+rem install all missing modules
+
+npm install express
```

Comment:
Instead of installing one module by one, why don't we do an npm install? It should work if it doesn't work something bad is happening.